Transbank/WebpayRest/Oneclick/Request/AuthorizeRequest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Net.Http;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Transbank.Webpay.Common;
+
+namespace Transbank.Webpay.Oneclick.Requests
+{
+    internal class AuthorizeRequest : BaseRequest
+    {
+
+        [JsonProperty("username")]
+        public string UserName { get; set; }
```

Comment:
Properties' access level should be equal or stricter than the Class access level